DESCRIPTION

```diff
@@ -26,6 +26,7 @@ Suggests:
     httr,
     jsonvalidate (>= 1.2.2),
     mockery,
+    openssl,
```

Comment:
for testing, e.g. https://github.com/vimc/orderly.server/pull/114/files#diff-7c334b153ab17686c8d7b6c765e5436d92345f5ea7144466931211b40809509aR754